template/template.go

```diff
@@ -28,7 +29,8 @@ func Apply(contents []byte, releaseInfo ReleaseInfo, variables map[string]string
 	buf := bytes.Buffer{}
 	t.Execute(&buf, map[string]interface{}{
 		"Values":  variables,
-		"Release": releaseInfo,
+		"Release": stackInfo,
```

Comment:
@joshwget what is the need by passing both Release and Stack given that they have the same value? Do we have some backwards compatibility to maintain here?